conf/modules.config

```
@@ -123,5 +123,21 @@ process {
         ]
         ext.prefix = { "${report}" }
     }
+
+    withName: ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES {
+        publishDir = [
+            path: { "${params.outdir}/antismash/databases/" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
+
+    withName: ANTISMASH_ANTISMASHLITE {
+        publishDir = [
+            path: { "${params.outdir}/antismash/antismashlite/" },
```

```suggestion
            path: { "${params.outdir}/antismash/antismashlite/" },
```

Even if the module is called antismashlite, I think we should just call it antismash or antismash/results, because antismashlite isn't really an 'official' tool so to say, just a 'hacky' variant. We don't want to confuse a user in this regard thinking it's a different tool that may ghave different output.